trifid/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "trifid.fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "trifid.name" . }}
```

Comment:
Please use the new form of labels: https://helm.sh/docs/topics/chart_best_practices/labels/#standard-labels

Reply:
Changed the labels to the new form (`helm create` uses the old labels...)